inference-engine/tests/functional/plugin/shared/include/single_layer_tests/activation_range.hpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
Why do you create a duplicate of activation tests? As i can see, we have similar tests cases in `activation.cpp`